package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "eslint-plugin-wingsui-rules",
```

Comment:
we should name them `harnessui-rules` instead of `wingsui-rules`

Reply:
Makes sense. Will change